QualityControl/lib/services/Status.service.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { Log } from '@aliceo2/web-ui';
+import { execSync } from 'node:child_process';
```

Comment:
Isn't the async version better to use in express route? 
```js
import {exec: execCallback} from 'node:child_process';
const exec = util.promisify(execCallback);
```

Reply:
Of course, thank you for pointing it out.
Updated